packages/appsync-emulator-serverless/__test__/example/serverless.yml

```diff
@@ -143,6 +153,11 @@ custom:
           lambdaFunctionArn: { Fn::GetAtt: [GraphqlLambdaFunction, Arn] }
           serviceRoleArn: { Fn::GetAtt: [AppSyncLambdaServiceRole, Arn] }
           functionName: graphql
+      - type: HTTP
+        name: TestHTTP
+        description: 'Test HTTP endpoint'
+        config:
+          endpoint: 'http://localhost:3000'
```

Comment:
hmmm, I think we should parameterize this value

Reply:
Does it matter with Nock? IIRC it intercepts the call.

Reply:
I'm not sure what it would mean to "parameterize" this.

Reply:
I think what @cbaron meant was allow this to be configured somehow to a different value. I don't think this is needed as nock will intercept all calls so there (if I understand correctly) does not even need to be a server running on port 3000 (or there can be with a conflict and it will still work just fine)

Reply:
Yes, nock does intercept the call, so 3000 should be fine.